stages/org.osbuild.update-crypto-policies

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
+"""
+Sets the policy applicable for the various cryptographic back-ends,
+such as SSL/TLS libraries.
+
+This stage calls `update-crypto-policies` to set the system's
+cryptographic policy.
+
+Notes:
+  - Requires 'chroot' in the buildroot.
+  - Runs the 'update-crypto-policies' script from the image in the chroot.
+"""
+
+
+import subprocess
+import sys
+
+from osbuild import api
+
+SCHEMA_2 = r"""
+"options": {
+  "additionalProperties": false,
+  "description": "Enables the system FIPS mode",
+  "required": [ "policy" ],
+  "properties": {
+    "policy": {
+      "type": "string",
```

Comment:
Would it make sense to have a stricter check what is allowed here? I.e. what are the possible values? Might be nice to guide the user with e.g. an `enum` in the schema about the possible values.

Reply:
Honestly, I am not sure, since the user can define his own policies the string can be anything.

Reply:
The good news is that setting a non-existent policy will fail, so an error won't be silently ignored.

Reply:
On the other hand, the schema can at least specify that the string can't be empty. IOW it would be good to specify at least `"minLenght": 1`.